service/orchestrator/toe.go

```diff
@@ -153,9 +156,32 @@ func (svc *Service) RemoveTargetOfEvaluation(_ context.Context, req *orchestrato
 		return nil, status.Errorf(codes.Internal, "database error: %v", err)
 	}
 
+	go svc.informToeHooks(ctx, nil, orchestrator.ToeStatus_TOE_REMOVE, nil)
 	return &emptypb.Empty{}, nil
 }
 
+// informToeHooks informs the registered hook functions about create, update or remove of the Target of Evaluation
+func (s *Service) informToeHooks(ctx context.Context, toe *orchestrator.TargetOfEvaluation, status orchestrator.ToeStatus_Status, err error) {
```

Comment:
Would it make sense to call this hook also from the update monitoringcontrol status?. Since in this case the "contents" / scope of the ToE has changed? I think this is needed for MEDINA, where we we need to find out if a control has changed to continuously monitored?